spk/comskip/Makefile

```diff
@@ -0,0 +1,38 @@
+SPK_NAME = comskip
+SPK_VERS = 0.81.089
+SPK_REV = 1
+SPK_ICON = src/comskip.gif
+
+DEPENDS = cross/ffmpeg cross/argtable cross/$(SPK_NAME)
+
+RELOAD_UI = yes
```

Comment:
If the package isn't startable, and doesn't have an icon to show in DSM's menu, this isn't needed. 
@cytec's example (and probably a whole heap of others) should be cleaned up at some point :)